### DISCUSSION TITLE
cd on quit

### DISCUSSION THREAD

**Discussion by praveenivp** [jarun/nnn#1835] [Q&A] [answered]
I want to `cd` to the current directory when i quit `nnn`.  I just added `NNN_TMPFILE` to my `.bashrc`. The file indeed has the `cd` command I want to run. How do I trigger it automatically. Should I have to launch `nnn` with some flags? 

my .bashrc
```
# nnn config
export NNN_TMPFILE='/tmp/.lastd'
export EDITOR=nano
```

**Comment by N-R-K** [comment, marked as answer]
Follow instructions at https://github.com/jarun/nnn/wiki/Basic-use-cases#configure-cd-on-quit

**Comment by praveenivp** [comment]
Thank you. Adding `n()` function in  the [quitcd bash script](https://github.com/jarun/nnn/blob/master/misc/quitcd/quitcd.bash_sh_zsh) to my `.bashrc` did the trick.

I feel it is overly complicated for an expected intuitive behavior! Thanks for the nice tool though:D

**Reply by N-R-K** [reply]
> I feel it is overly complicated for an expected intuitive behavior!

It's not possible to do it otherwise. A process (`nnn` in this case) cannot arbitrarily change the working directory of another process (the parent shell).

**Reply by praveenivp** [reply]
Now I see. Thanks for the clarification.
